src/pages/candidates/Candidates.jsx

```diff
@@ -43,6 +59,13 @@ const Candidates = () => {
     dispatch(fetchCandidateList(requestBody));
   }, []);
 
+  if (!listOfCandidates.length) return <ProgressIndicator />;
```

Comment:
what if there are no candidates on an internship yet? Will it just keep 'loadng'?

Reply:
done